src/no-wallaby-comment.spec.ts

```diff
@@ -0,0 +1,45 @@
+// no-wallaby-comment.spec.ts
+
+/*
+ * Copyright (c) 2022-2023 Check Digit, LLC
+ *
+ * This code is licensed under the MIT license (see LICENSE.txt for details).
+ */
+import { RuleTester } from 'eslint';
+
+import rule from './no-wallaby-comment';
+
+const LINE_WITH_NO_COMMENTS = `const NOT_A_SECRET = "A template that isn't a secret.";`;
+
+const LINE_WITH_COMMENT = `const NOT_A_SECRET = "A template that isn't a secret."; // ?`;
```

Comment:
there's no test for `file.only`, and also i think there should be single comment tests for the various varieties (e.g. `// ?.` etc)